app/models/permission.rb

```diff
@@ -2,15 +2,17 @@
 #
 # Table name: permissions
 #
-#  id            :bigint           not null, primary key
-#  can_edit      :boolean
-#  can_edit_role :boolean
-#  can_lock_user :boolean
-#  can_read      :boolean
-#  can_set_role  :boolean
-#  created_at    :datetime         not null
-#  updated_at    :datetime         not null
-#  role_id       :bigint           not null
+#  id                        :bigint           not null, primary key
+#  can_edit_entities         :boolean
+#  can_edit_movie_and_person :boolean
+#  can_edit_role             :boolean
+#  can_lock_user             :boolean
+#  can_read_entities         :boolean
+#  can_read_movie_and_person :boolean
+#  can_set_role              :boolean
+#  created_at                :datetime         not null
+#  updated_at                :datetime         not null
+#  role_id                   :bigint           not null
```

Comment:
looks strange. Please try use git better

Reply:
but it annotates comments, it aligns them, not git

Reply:
exactly! But it is NOT the best practice. YOU should write code  satisfied to linters requirements